### DISCUSSION TITLE
科技爱好者周刊（第 252 期）：互联网创业变难了 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#14] [General]
# 科技爱好者周刊（第 252 期）：互联网创业变难了 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-252

**Comment by terminalqo** [comment]
可以把捐赠码放到顶上~

**Reply by plantree** [reply]
感谢您的提醒！这是捐给[免费午餐基金](http://www.mianfeiwucan.org/)的，大家文章看累了，愿意的话可以支持一下。

**Comment by zhu-hongwei** [comment]
every thing is cool
